docs/en/administration/monitoring.md

```diff
@@ -5,24 +5,24 @@ sidebar_position: 3
 
 # Monitoring and Data Visualization
 
-As a distributed file system hosting massive data storage, we need to have the ability to visualize the status changes of the entire system in terms of capacity, files, CPU load, disk IO, cache, etc. JuiceFS provides real-time status data to the outside world through a Prometheus-based API. Just add it to your own Prometheus Server to scrape and store time series data, and then easily visualize and monitor the JucieFS file system with tools like Grafana.
+As a distributed file system hosting massive data storage, it is important for users to directly view the status changes of the entire system in terms of capacity, files, CPU load, disk IO, cache, etc. JuiceFS provides real-time status data externally through the Prometheus-oriented API to achieve the visualization of JuiceFS monitoring with ease, and you only need to expose it to your own Prometheus Server to visualize time series data with tools like Grafana.
 
 ## Get started
 
-It is assumed here that you build Prometheus Server, Grafana and JuiceFS clients all running on the same host.
+It is assumed here that Prometheus Server, Grafana and JuiceFS clients are all running on the same host, in which
 
 - **Prometheus Server**: Scrapes and stores the time series data of various metrics. For installation, please refer to the [official documentation](https://prometheus.io/docs/introduction/first_steps/).
-- **Grafana**: Loading and visualizing the time series data from Prometheus. For installation, please refer to the [official documentation](https://grafana.com/docs/grafana/latest/installation/).
+- **Grafana**: Loads and visualizes the time series data from Prometheus. For installation, please refer to the [official documentation](https://grafana.com/docs/grafana/latest/installation/).
 
 ### Ⅰ. Access to real-time data
 
-JuiceFS outputs metrics data via an API of type [Prometheus](https://prometheus.io). After the file system is mounted, the live monitoring data output from the client is available by default address `http://localhost:9567/metrics`.
+JuiceFS outputs metrics data via a [Prometheus](https://prometheus.io)-oriented API. After the file system is mounted, the real-time monitoring data output from the client can be obtained from the default address `http://localhost:9567/metrics`.
 
 ![](../images/prometheus-client-data.jpg)
 
 ### Ⅱ. Add API to Prometheus Server
 
-Edit the [configuration file](https://prometheus.io/docs/prometheus/latest/configuration/configuration) of Prometheus, add a new job and point to the API address of JuiceFS, e.g.
+Edit the [configuration file](https://prometheus.io/docs/prometheus/latest/configuration/configuration) of Prometheus, add a new job and point it to the API address of JuiceFS, e.g.
```

Comment:
Edit the [Prometheus configuration file](https://prometheus.io/docs/prometheus/latest/configuration/configuration), add a new job and point it to the API address of JuiceFS, e.g.